[2021-04-23 16:05:40,496] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T12:00:00+00:00 [queued]>
[2021-04-23 16:05:40,568] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T12:00:00+00:00 [queued]>
[2021-04-23 16:05:40,570] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 16:05:40,571] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2021-04-23 16:05:40,573] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 16:05:40,603] {taskinstance.py:1063} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc> on 2021-04-23T12:00:00+00:00
[2021-04-23 16:05:40,633] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'create_cluster_dataproc', '2021-04-23T12:00:00+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmphky7luhw', '--error-file', '/tmp/tmp5p4rzg1q']
[2021-04-23 16:05:40,654] {standard_task_runner.py:77} INFO - Job 11: Subtask create_cluster_dataproc
[2021-04-23 16:05:40,620] {standard_task_runner.py:52} INFO - Started process 39 to run task
[2021-04-23 16:05:40,989] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T12:00:00+00:00 [running]> on host 760ffd3d790b
[2021-04-23 16:05:41,447] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-23T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-23T12:00:00+00:00
[2021-04-23 16:05:41,451] {dataproc.py:603} INFO - Creating cluster: ephemeral-recent-played-spark-cluster-20210423
[2021-04-23 16:09:50,381] {dataproc.py:547} INFO - Cluster created.
[2021-04-23 16:09:50,645] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=create_cluster_dataproc, execution_date=20210423T120000, start_date=20210423T160540, end_date=20210423T160950
[2021-04-23 16:09:50,866] {local_task_job.py:146} INFO - Task exited with return code 1
